docs/guides/use_checkpointing.ipynb

```diff
@@ -5,17 +5,25 @@
    "id": "6e9134fa",
```

Comment:
<p>add pip quiet option to silent install warnings..... why not add this install at the very top with the rest??</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/2688/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>